daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/InfosetImpl.scala

```diff
@@ -383,7 +383,7 @@ sealed abstract class LengthState(ie: DIElement) {
   var maybeStartPos0bInBits: MaybeULong = MaybeULong.Nope
   var maybeEndDataOutputStream: Maybe[DataOutputStream] = Nope
   var maybeEndPos0bInBits: MaybeULong = MaybeULong.Nope
-  var maybeComputedLength: MaybeULong = MaybeULong.Nope
+  var maybeComputedLength: MaybeULong = MaybeULong.Nope // Units are bits
```

Comment:
Could rename `maybeComputedLength` to `maybeComputedLenInBits`, then wouldn't need this comment.